csrankings-i.csv

```diff
@@ -392,6 +392,7 @@ Ivor P. Page,University of Texas at Dallas,http://www.utdallas.edu/~ivor,NOSCHOL
 Ivor T. A. Spence,Queen's University Belfast,https://www.qub.ac.uk/schools/eeecs/Connect/Staff/BusinessCard/?name=i.spence,pN5KXz4AAAAJ
 Ivor W. Tsang,University of Technology Sydney,https://www.uts.edu.au/staff/ivor.tsang,rJMOlVsAAAAJ
 Ivor Wai-Hung Tsang,University of Technology Sydney,https://www.uts.edu.au/staff/ivor.tsang,rJMOlVsAAAAJ
+Ivy Bo Peng,KTH Royal Institute of Technology,https://ibpeng.github.io,wqC4XFoAAAAJ

```

Comment:
Affiliation still says LLNL affiliation on web page, LinkedIn, and Google Scholar. Please point to something documenting this affiliation.

Reply:
Please address this issue ASAP; I am closing old PRs by the end of this week.

Reply:
This is the KTH profile: https://www.kth.se/profile/bopeng